api/core/model_runtime/model_providers/anthropic/llm/llm.py

```diff
@@ -526,17 +526,19 @@ def _convert_prompt_messages(self, prompt_messages: Sequence[PromptMessage]) ->
                                 }
                                 sub_messages.append(sub_message_dict)
                             elif isinstance(message_content, DocumentPromptMessageContent):
-                                if message_content.mime_type != "application/pdf":
+                                data_split = message_content.data.split(";base64,")
+                                mime_type = data_split[0].replace("data:", "")
+                                base64_data = data_split[1]
```

Comment:
I think we need some helpers or properties in PromptMessageContent model to extract mime_type and b64_data.

Reply:
sounds good !  It's a better way!

Reply:
I make changes in another PR https://github.com/langgenius/dify/pull/11721 , this PR plan to only modify related with gemini models